tool/jt.rb

```diff
@@ -716,6 +716,10 @@ def test_mri(*args)
     if args.delete('--openssl')
       include_pattern = "#{TRUFFLERUBY_DIR}/test/mri/tests/openssl/test_*.rb"
       exclude_file = "#{TRUFFLERUBY_DIR}/test/mri/openssl.exclude"
+    elsif args.delete('--cext')
```

Comment:
I've renamed this to `--syslog`. I think a consolidated `--cext` would be a good idea. Could this be done in a separate PR?